power_grid_model_c/power_grid_model/include/power_grid_model/math_solver/iterative_linear_se_solver.hpp

```diff
@@ -695,11 +765,13 @@ template <bool sym> class IterativeLinearSESolver {
                         for (IntS const measured_side : std::array<IntS, 2>{0, 1}) {
                             // has measurement
                             if (std::invoke(has_branch_[measured_side], measured_value, obj)) {
-                                // G += Y{side, b0}^H * Y{side, b1} / variance
+                                // G += Y{side, b0}^H * Y{side, b1} * (variance^-1)
```

Comment:
This should be `Y{side, b0}^H * (variance^-1) * Y{side, b1}`

Reply:
you're right 🤦 how did i not catch this